doc8/Dockerfile

```
@@ -0,0 +1,11 @@
+FROM registry.fedoraproject.org/fedora:35
+
+RUN dnf -y --nodocs  --setopt=install_weak_deps=False \
+--disablerepo=fedora-cisco-openh264 \
+--disablerepo=fedora-modular \
+--disablerepo=updates-modular \
+install python3-pip python3-setuptools python3-wheel git && \
+dnf clean all && \
+pip3 --no-cache-dir install "doc8==0.9.1"
+
+ENTRYPOINT ["doc8", "--max-line-length 100"]
```

```suggestion
ENTRYPOINT ["doc8", "--max-line-length=100"]
```
or
```suggestion
ENTRYPOINT ["doc8", "--max-line-length", "100"]
```
The current syntax gets interpreted as `doc8 '--max-line-length 100'` and you get `OSError: Invalid path: --max-line-length 100`